src/main/webapp/map.html

```diff
@@ -0,0 +1,125 @@
+                elementType: 'labels.text.fill',
+                stylers: [{color: '#d59563'}]
+            },
+            {
+                featureType: 'poi',
+                elementType: 'labels.text.fill',
+                stylers: [{color: '#d59563'}]
+            },
+            {
+                featureType: 'poi.park',
+                elementType: 'geometry',
+                stylers: [{color: '#263c3f'}]
+            },
+            {
+                featureType: 'poi.park',
+                elementType: 'labels.text.fill',
+                stylers: [{color: '#6b9a76'}]
+            },
+            {
+              featureType: 'road',
+              elementType: 'geometry',
+              stylers: [{color: '#38414e'}]
+            },
+            {
+                featureType: 'road',
+                elementType: 'geometry.stroke',
+                stylers: [{color: '#212a37'}]
+            },
+            {
+              featureType: 'road',
+              elementType: 'labels.text.fill',
+              stylers: [{color: '#9ca5b3'}]
+            },
+            {
+                featureType: 'road.highway',
+                elementType: 'geometry',
+                stylers: [{color: '#746855'}]
+            },
+            {
+                featureType: 'road.highway',
+                elementType: 'geometry.stroke',
+                stylers: [{color: '#1f2835'}]
+            },
+            {
+                featureType: 'road.highway',
+                elementType: 'labels.text.fill',
+                stylers: [{color: '#f3d19c'}]
+            },
+            {
+                featureType: 'transit',
+                elementType: 'geometry',
+                stylers: [{color: '#2f3948'}]
+            },
+            {
+                featureType: 'transit.station',
+                elementType: 'labels.text.fill',
+                stylers: [{color: '#d59563'}]
+            },
+            {
+                featureType: 'water',
+                elementType: 'geometry',
+                stylers: [{color: '#17263c'}]
+            },
+            {
+                featureType: 'water',
+                elementType: 'labels.text.fill',
+                stylers: [{color: '#515c6d'}]
+            },
+            {
+                featureType: 'water',
+                elementType: 'labels.text.stroke',
+                stylers: [{color: '#17263c'}]
+                }]
+            });
+
+            //Adds a marker at Stan the T-Rex
+            const trexMarker = new google.maps.Marker({
+                position: {lat: 37.421903, lng: -122.084674},
+                map: map,
+                title: 'Stan the T-Rex'
+            });
+
+            //info window for the T-rex
+            var trexInfoWindow = new google.maps.InfoWindow({
+                content: 'This is Stan, the T-Rex statue.'
+            });
+            trexInfoWindow.open(map, trexMarker);
+
+
+        }
+    </script>
+    <style>
```

Comment:
This style section can also be in a css file. I think this could be either in a map.css file or the main.css file depending on how the style works. (Like, if you want all maps throughout the whole site to have these style attributes put it in main.css, if you want this styling to be specific to the #map <div> on this page you can make a map.css file and plug this in there :)

Reply:
Ok, would I just copy and paste the code into map.css. Where would I put map.css?